atoma-state/src/state_manager.rs

```diff
@@ -260,12 +255,23 @@ impl AtomaState {
     pub async fn get_stacks_for_model(&self, model: &str, free_units: i64) -> Result<Vec<Stack>> {
```

Comment:
Can we add unit tests for the update part ?

Reply:
The problem is that for testing you will need to run postgres. That's why there are no tests in this file.